Code completion in completion1.xml for marginBottom="50"^ selecting  android:maxEms:
@@ -12 +12
-         android:layout_marginBottom="50"^
+         android:layout_marginBottom="50" android:maxEms="^"
